src/app/components/SearchResults/SearchResultsContainer.jsx

```diff
@@ -13,65 +13,73 @@ import Actions from '../../actions/Actions';
 import appConfig from '../../data/appConfig';
 
 // Renders the ResultsList containing the search results and the Pagination component
-const SearchResultsContainer = (props) => {
-  const {
-    searchResults,
-    searchKeywords,
-    page,
-  } = props;
+class SearchResultsContainer extends React.Component {
```

Comment:
why are we making this a class component? It doesn't seem to have state

Reply:
I somehow missed this from the docs the last time I looked into using `context` in a functional component: https://reactjs.org/docs/legacy-context.html#referencing-context-in-stateless-function-components

Reverting now